AgileFront/src/app/app-routing.module.ts

```diff
@@ -22,11 +24,16 @@ const routes: Routes = [
     component: CreatePostComponent,
     canActivate: [authGuard],
   },
+  {
+    path: 'post',
+    component: ProjectComponent,
+    canActivate: [authGuard],
```

Comment:
We'll might want to remove tha guard, but add protection to redirect the user to the login page if he try to like or comment a post

Reply:
I'll implement it with the like button no worries